docs/internal/continuous-profiler.md

```diff
@@ -0,0 +1,255 @@
+# Continuous profiler
+
+> [!IMPORTANT]  
+> Continuous profiler is an experimental feature. It will be subject to change,
+> when <https://github.com/open-telemetry/oteps/pull/239> or <https://github.com/open-telemetry/oteps/pull/237>
+> will be ready and merged.
+
+## Thread sampling
+
+Thread sampling can be enabled by the custom plugin.
+Plugin is responsible for parsing dense, data format and exporting it in
+the appropriate format.
+
+### How does the thread sampler work?
+
+The profiler leverages [.NET profiling](https://docs.microsoft.com/en-us/dotnet/framework/unmanaged-api/profiling/)
+to perform periodic call stack sampling. For every sampling period, the runtime
+ suspended and the samples for all managed thread are saved into the buffer,
+ then the runtime resumes.
+
+The separate managed-thread is processing data from the buffer and export it
+the way defined by the plugin.
+
+To make the process more efficient, the sampler uses two independent buffers to
+store samples alternatively.
+
+### Requirements
+
+* .NET 6.0 or higher (`ICorProfilerInfo12` available in runtime). Technically
+  it could be `ICorProfiler10`, and .NET Core 3.1/.NET 5.0, but these versions
+  are not supported by OpenTelemetry .NET AutoInstrumentation.
+* .NET Framework is not supported. `ICorProfiler10` nor `ICorProfiler12`
+  are not available in .NET Fx.
+
+### Enable the profiler
+
+Implement custom plugin.
+
+### Configuration settings by plugin
+
+* `threadSamplingEnabled = true;`
+* `var threadSamplingInterval = 10000u;` // in ms. Splunk is using 10000.
+  There is no possibility to set value lower than 1000.
+* `var exportInterval = TimeSpan.FromMilliseconds(500);` // Interval to read data
+  from buffers and call exporter, common for Thread and Allocation sampling
+* `object continuousProfilerExporter = new ConsoleExporter();` // Exporter,
+  common for Thread and Allocation sampling
```

Comment:
```suggestion
### Configuration defaults

* `threadSamplingEnabled = true;`: Enables thread sampling.
* `var threadSamplingInterval = 10000u;`: Sampling interval, in milliseconds. Lowest possible value is 1000.
* `var exportInterval = TimeSpan.FromMilliseconds(500);`: Interval for reading the data from buffers and call the exporter. This setting is common for both thread and allocation sampling.
* `object continuousProfilerExporter = new ConsoleExporter();`: Exporter to be used for both thread and allocation sampling.
```

It's defaults as set by the plugin, right?

Reply:
Plugin author manage the default values. There is no guards in the OTel AutoInsturmentation